test/server/cards/02_SHD/upgrades/PublicEnemy.spec.ts

```diff
@@ -0,0 +1,52 @@
+describe('Public Enemy', function() {
+    integration(function(contextRef) {
+        describe('Public Enemy\'s Bounty ability', function() {
+            it('should give a unit a shield', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [
+                            { card: 'battlefield-marine' },
```

Comment:
If you aren't adding any special properties like damage or upgrades you can just provide these as string names. E.g.:

```typescript
groundArena: ['battlefield-marine', 'atst']
```

Can mix and match both styles as needed